### PR TITLE
Fix display issue with registry section display of checkboxes and rad…

### PR DESCRIPTION
**Pull request by alintreznai**
This MR fixes an issue with incorrect display of checkboxes and radio buttons when a  registry configured section is displayed in the patient forms.

### CODE REVIEW

File: rdrf/rdrf/templatetags/is_checkbox.py [hunk 1]
```diff
@@ -13,7 +14,7 @@ def is_checkbox(element):
     if hasattr(element, "field"):
         field = element.field
 
-    if (field.widget.__class__.__name__ == CheckboxInput().__class__.__name__):
+    field_class_name = field.widget.__class__.__name__
+    checkbox_class_names = (CheckboxInput().__class__.__name__, CheckboxSelectMultiple().__class__.__name__, )
 
-        return True
-    return False
+    return field_class_name in checkbox_class_names
```
Comment (sztamas) [inline rdrf/rdrf/templatetags/is_checkbox.py]: Doesn't a check based on `field.widget.input_type == 'checkbox'` look better here?

File: rdrf/rdrf/templatetags/is_radio.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.forms.widgets import RadioSelect
+
+register = template.Library()
+
+
+@register.filter(name='is_radio')
+def is_radio(element):
+    if hasattr(element, "field"):
+        field = element.field
```
Comment (sztamas) [inline rdrf/rdrf/templatetags/is_radio.py]: Shouldn't we shortcut the function (return False) if there is no field on element?

File: rdrf/rdrf/templates/rdrf_cdes/patient_edit.html [hunk 3]
```diff
@@ -305,24 +306,20 @@
                                                                         {% endif %}
                                                                     </label>
                                                                     <div class="col-sm-9">
-                                                                        {% if not element|is_checkbox %}
+                                                                        {% if not element|is_checkbox and not element|is_radio %}
```
Comment (sztamas) [inline rdrf/rdrf/templates/rdrf_cdes/patient_edit.html]: Why are these ifs needed at all? Can't we just always use `element` and let these take care of setting the `form-control` CSS class?

https://github.com/eresearchqut/trrf/blob/ad581bb3acd6b0f2dc836ef910af6e5d5805944c/rdrf/rdrf/templates/rdrf_cdes/patient_edit.html#L124

**Review by sztamas — COMMENTED** [review]
Left some inline comments.

Additionally, the CSS style of the `<ul>` of the checkbox should be styled so that `list-style-type` is none and there is no margin and padding on the left.

File: rdrf/rdrf/templates/rdrf_cdes/patient_edit.html (hunk 3 above)
Comment (alintreznai) [reply rdrf/rdrf/templates/rdrf_cdes/patient_edit.html]: Indeed. Did not notice this.

File: rdrf/rdrf/templatetags/is_checkbox.py (hunk 1 above)
Comment (alintreznai) [reply rdrf/rdrf/templatetags/is_checkbox.py]: Updated.

File: rdrf/rdrf/templatetags/is_radio.py (hunk 2 above)
Comment (alintreznai) [reply rdrf/rdrf/templatetags/is_radio.py]: Updated.

**Review by sztamas — APPROVED** [review]

**Pull request merged**